pkg/storaged/storage-controls.jsx

```diff
@@ -118,7 +118,7 @@ export class StorageLink extends React.Component {
                             content={(excuse) => (
                                 <button onClick={checked(this.props.onClick)}
                                         style={excuse ? { pointerEvents: 'none' } : null}
-                                        className={"link-button ct-form-relax" + (excuse ? " disabled" : "")}>
```

Comment:
Setting `disabled` to `excuse` seems a bit weird as it is usually some string or some dom element. It should just check if `excuse` is defined or not. (I think that `!!excuse` should do the trick).

Reply:
Should I update all `disabled={excuse}`s, there are 4 ocurrencies in that file?

Reply:
Yes, please